docs/generated/oc_by_example_content.adoc

```diff
@@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
 ====
 
 
+== oc secrets new-sourcecfg
+Create a new source secret
```

Comment:
Yeah those are fine.  Whatever types we pick should try to be consistent
though so we need to propose those upstream.

On Sep 9, 2015, at 10:56 AM, Jakub Hadvig <notifications@github.com> wrote:

In docs/generated/oc_by_example_content.adoc
<https://github.com/openshift/origin/pull/4433#discussion_r39053055>:

> @@ -776,6 +776,28 @@ Create a new secret based on a key file or on files within a directory
>  ====
>
>
> +== oc secrets new-sourcecfg
> +Create a new source secret

@smarterclayton <https://github.com/smarterclayton> @csrwng
<https://github.com/csrwng> what about:

   - new-basicauth
   - new-sshauth

?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4433/files#r39053055>.